crates/uv-resolver/src/lock/mod.rs

```diff
@@ -2837,6 +2837,35 @@ struct PackageMetadata {
     dependency_groups: BTreeMap<GroupName, BTreeSet<Requirement>>,
 }
 
+impl PackageMetadata {
+    fn unwire(self, requires_python: &RequiresPython) -> Result<PackageMetadata, LockError> {
```

Comment:
Should we have a PackageMetadataWire type? Is it problematic that we're serializing raw Requirements and not the SimplifiedMarkerTree stuff DependencyWire has?